### PR TITLE
docs: Add required Next.js version to README

### PR DESCRIPTION
**Pull request by valuegrammer**
i wax trying to use next-i18next with a project i was working on but couldn't quite get locale subpaths working properly 

after some investigation it turns out i had nextjs version 9.3 which doesnt support rewrites and that took me quite a while to notice

So i added extra information to the read me to prevent others from falling in the same pitfall

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ After creating and exporting your `NextI18Next` instance, you need to take the f
 
 1. Create an `_app.js` file inside your `pages` directory, and wrap it with the `NextI18Next.appWithTranslation` higher order component (HOC). You can see this approach in the [examples/simple/pages/_app.js](./examples/simple/pages/_app.js).
 Your app component must either extend `App` if it's a class component or define a `getInitialProps` if it's a functional component [(explanation here)](https://github.com/isaachinman/next-i18next/issues/615#issuecomment-575578375).
-2. Create a `next.config.js` file inside your root directory if you want to use locale subpaths. You can see this approach in the [examples/simple/next.config.js](./examples/simple/next.config.js).
+2. Create a `next.config.js` file inside your root directory if you want to use locale subpaths. You can see this approach in the [examples/simple/next.config.js](./examples/simple/next.config.js) **make sure you have next.js version 9.5 or newer**.
```
Comment (juzhiyuan) [inline README.md:72]: `Next.js 9.5 or higher` would be better

Comment (valuegrammer) [reply README.md:72]: Yeah that's right i'll change it when i get on my computer

**Comment by valuegrammer** [comment]
Alright made it more concise @juzhiyuan

**Comment by juzhiyuan** [comment]
Just waiting for Admin's review :D

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ After creating and exporting your `NextI18Next` instance, you need to take the f
 
 1. Create an `_app.js` file inside your `pages` directory, and wrap it with the `NextI18Next.appWithTranslation` higher order component (HOC). You can see this approach in the [examples/simple/pages/_app.js](./examples/simple/pages/_app.js).
 Your app component must either extend `App` if it's a class component or define a `getInitialProps` if it's a functional component [(explanation here)](https://github.com/isaachinman/next-i18next/issues/615#issuecomment-575578375).
-2. Create a `next.config.js` file inside your root directory if you want to use locale subpaths. You can see this approach in the [examples/simple/next.config.js](./examples/simple/next.config.js).
+2. Create a `next.config.js` file inside your root directory if you want to use locale subpaths. You can see this approach in the [examples/simple/next.config.js](./examples/simple/next.config.js) **Next.js 9.5 or newer required**.
```
Comment (isaachinman) [inline README.md:72]: Please change to `(Next.js v9.5+ required)`.

**Review by isaachinman — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (valuegrammer) [reply README.md]: Done.

Comment (juzhiyuan) [reply README.md]: v9.5+ please

Comment (valuegrammer) [reply README.md]: that's what i did?
Am i getting you right i'm pretty sure that's what i did

Comment (juzhiyuan) [reply README.md]: image

Comment (valuegrammer) [reply README.md]: https://github.com/valuegrammer/next-i18next
that's what it says.... i'm confused

Comment (isaachinman) [reply README.md]: I'm asking you to literally change the text to read `(Next.js v9.5+ required)`.

**Review by isaachinman — APPROVED** [review]

**Pull request merged**